pkg/build/builder/common.go

```diff
@@ -30,3 +36,47 @@ func buildInfo(build *api.Build) []KeyValue {
 	}
 	return kv
 }
+
+// setHTTPProxy sets the system's environment variables to define the HTTP and
+// HTTPS proxies to be used by commands that respect those variables, e.g., git
+// clone performed to fetch source code to be built. It returns the original
+// values of all environment variables that were set.
+func setHTTPProxy(httpProxy, httpsProxy string) map[string]string {
+	originalProxies := make(map[string]string, 4)
+	if httpProxy != "" {
+		glog.V(2).Infof("Setting HTTP_PROXY to %s", httpProxy)
```

Comment:
defaults to the loglevel openshift was started with.  after that it uses the env variable you set on the buildconfig:
https://docs.openshift.org/latest/dev_guide/builds.html#accessing-build-logs